### PR TITLE
base_sm6375: Update defconfig

### PR DESCRIPTION
**Pull request by bartcubbins**
Remove unnecessary entries, distribute some entriees by platform fragments and enable the missing ones.

Also bring back some of the changes that were erroneously deleted in https://github.com/sonyxperiadev/kernel-defconfig/commit/9a2f9fbba216d10615ef1c87ead6031fc9e659a1

### CODE REVIEW

File: base_zambezi_pdx235_defconfig [hunk 1]
```diff
@@ -1,11 +1,4 @@
 CONFIG_ARCH_SONY_ZAMBEZI=y
 CONFIG_MACH_SONY_PDX235=y
-# CONFIG_NFC_QTI_I2C is not set
-# CONFIG_FP_ET613 is not set
-CONFIG_FP_EC617=y
```
Comment (jerpelea) [inline base_zambezi_pdx235_defconfig:5]: please don't remove the config for the missing drivers

File: base_zambezi_pdx235_defconfig [hunk 2]
```diff
@@ -1,11 +1,4 @@
...
-CONFIG_TOUCHSCREEN_FTS=y
-CONFIG_TOUCHSCREEN_FTS_DIRECTORY="focaltech_touch"
-# CONFIG_TOUCHSCREEN_SEC_TS is not set
-CONFIG_SMB1398_CHARGER=y
-CONFIG_SMB1355_SLAVE_CHARGER=y
-CONFIG_LEDS_AW2016=y
```
Comment (jerpelea) [inline base_zambezi_pdx235_defconfig:11]: please don't remove the config for the missing drivers

File: base_zambezi_pdx235_defconfig [hunk 3]
```diff
@@ -1,11 +1,4 @@
 CONFIG_ARCH_SONY_ZAMBEZI=y
 CONFIG_MACH_SONY_PDX235=y
-# CONFIG_NFC_QTI_I2C is not set
-# CONFIG_FP_ET613 is not set
-CONFIG_FP_EC617=y
-CONFIG_TOUCHSCREEN_FTS=y
```
Comment (jerpelea) [inline base_zambezi_pdx235_defconfig:6]: please don't remove the config for the missing drivers

File: base_zambezi_pdx235_defconfig (hunk 2 above)
Comment (bartcubbins) [reply base_zambezi_pdx235_defconfig:11]: @jerpelea Zambezi uses QG+SMB5 combo, other entries either have alternative names (e.g. stock: CONFIG_FP_EC617 -> sodp: CONFIG_SENSORS_ET51X) or not used at all (CONFIG_LEDS_AW2016, PDX235 has no LED). they were not blindly removed

**Pull request merged**